### DISCUSSION TITLE
Milestones

### DISCUSSION THREAD

**Discussion by MartinSchmidt** [project-origin/registry#49] [General]
I think we should have a discussion about what should be included in our milestones, so that we set ourselves some goals what features we create when, and that users can follow:
[milestones](https://github.com/project-origin/registry/milestones)

I have created two milestones, to try and highlight the picture I have in my head, but i really think we should have an open discussion about the topic.

- [Alpha milestone](https://github.com/project-origin/registry/milestone/1)
- [Beta milestone](https://github.com/project-origin/registry/milestone/2)

**Comment by AndreAlnor** [comment]
@MartinSchmidt and I had a get together yesterday where we denied a new set of rules for milestones and versioning. We close this topic and describe this in a seperate doc
